utils/hints_test.go

```diff
@@ -227,6 +227,79 @@ func TestGenerateHints(t *testing.T) {
 			},
 			expectedIncorrectHints: 0,
 		},
+		// Scenarios being tested:
+		// have co.elastic.hints/package set.
+		// Define multiple co.elastic.hints/data_streams and also specific configuration for each one
+		// Typo errors introduced for "co.elastic.hints/access.streams" and "co.elastic.hints/error.streams"
+		{
+			name: "Metrics_apache_package_and_specific_config_per_datastream",
+			annotations: map[string]string{
+				"co.elastic.hints/package":        "apache",
+				"co.elastic.hints/data_streams":   "access,error",
+				"co.elastic.hints/access.period":  "5m",
+				"co.elastic.hints/access.streams": "stdout", // On purpose this added with typo
```

Comment:
I'm having a hard time understanding the typo here. As it is a test, it would be easier to understand if the typo is very clear.

My suggestion is that instead of using a common mistake, like instead of using 'stream'/'streams', you can use 'stream'/'streamsssss'. Something that even on a quick glance anyone can notice why it is a typo.

Reply:
The same applies to the other ones on this PR.

Reply:
@belimawr can you please have another look?

Reply:
Sorry for the delay @gizas for some reason I didn't get the email for this ping :/